src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -286,18 +286,20 @@ MetadataCreateCommands(void)
 		DistTableCacheEntry *cacheEntry =
 			(DistTableCacheEntry *) lfirst(distributedTableCell);
 		Oid relationId = cacheEntry->relationId;
+		ObjectAddress tableAddress = { 0 };
 
 		List *workerSequenceDDLCommands = SequenceDDLCommandsForTable(relationId);
 		List *ddlCommandList = GetTableDDLEvents(relationId, includeSequenceDefaults);
 		char *tableOwnerResetCommand = TableOwnerResetCommand(relationId);
 
 		/*
-		 * Ensure schema exists on each worker node. We can not run this function
-		 * transactionally, since we may create shards over separate sessions and
-		 * shard creation depends on the schema being present and visible from all
-		 * sessions.
+		 * distributed tables might have dependencies on different objects, since we
```

Comment:
Minor convention note in general: Typically multi-line comments starts with a capital letter. So, might be good to stick to that.